test/test-fgets-eof.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright © 2024 HanaMAshour
```

Comment:
Do you want to change the name used here?

Reply:
I changed the copyright and it is fixed now